src/routes/events.ts

```diff
@@ -63,5 +63,6 @@ export default async function events(fastify: FastifyInstance) {
     req.socket.on('close', () => {
       uploadManager.removeListener('refresh_table_row', placeChangeListener);
     });
+    
```

Comment:
can you revert these white space changes?  They just add clutter to the PR

Reply:
this file `events.ts` should not be in this PR because you just made white space changes - please revert